pkg/builds/cosa_v1.go

```diff
@@ -129,9 +129,6 @@ type Cloudartifact struct {
 
 type Extensions struct {
 	Manifest       map[string]interface{} `json:"manifest"`
-	Path           string                 `json:"path"`
-	RpmOstreeState string                 `json:"rpm-ostree-state"`
-	Sha256         string                 `json:"sha256"`
```

Comment:
I agree that I don't think we need these any longer. My only concern is do we somehow want to maintain backward compatibility in case we encounter this (i.e. using current tools to look at an older build).

Reply:
We would have already noticed this since no extensions were being created at all. I suggest backporting until the point where we stopped generating extensions in meta.json.

Reply:
Not sure I understand.. but let me ask a different question..

If you do bake this PR into a COSA image and then try to parse (i.e. buildfetch and `cosa meta --build=415.92.202501281917-0 --get extensions`) does it barf on you?

Reply:
right. I was just worried that the schema would error if there were components in there that it didn't expect.

Reply:
It works because the schema doesn't disallow additional properties. So the fields are essentially "unregistered". I think instead of this, we should make those properties optional. That way we still have type modelling for them. This could be useful in the future for GC.